install.sh

```diff
@@ -161,5 +161,12 @@ echo ""
 echo "Once running, access the Dispatch UI at:"
 echo ""
 echo "  http://localhost:8000/default/auth/register"
+echo ""
+echo "After registering, run the command below to get admin rights for the user you just registered:"
+echo ""
+echo "  docker exec -it dispatch_web_1 bash -c 'dispatch user update --role Admin --organization name-of-the-organization email-address-of-registered-user'"
+echo ""
```

Comment:
Since this is the first user, I think it would be more appropriate to give it `Owner` role permissions.
```suggestion
echo "After registering, run the command below to get owner rights for the user you just registered:"
echo ""
echo "  docker exec -it dispatch_web_1 bash -c 'dispatch user update --role Owner --organization name-of-the-organization email-address-of-registered-user'"
echo ""
```